packages/components/src/InputPassword/InputPassword.mdx

```diff
@@ -0,0 +1,63 @@
+---
+name: InputPassword
+menu: Components
+route: /components/input-password
+---
+
+import { Playground, Props } from "docz";
+import { ComponentStatus } from "@jobber/docx";
+import { useState } from "react";
+import { InputPassword } from ".";
+
+# Input Password
+
+<ComponentStatus stage="ready" responsive="yes" accessible="yes" />
+
+InputPassword is used to display a secure password input field.
+
+```ts
+import { InputPassword } from "@jobber/components/InputPassword";
+```
+
+<Playground>
+  <InputPassword defaultValue="password" placeholder="Password" />
+</Playground>
+
+## Props
+
+<Props of={InputPassword} />
```

Comment:
What is this line supposed to be doing? I see the header (line 27) but not sure what should be displayed here...

Reply:
https://www.docz.site/docs/built-in-components#component-props

Reply:
Thanks... the `npm run clean` fixed it locally.

Reply:
Discussed this in chat. All resolved I think.